src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/ElasticsearchCaseSearchRequestSecurity.java

```diff
@@ -44,6 +67,23 @@ private String addFiltersToQuery(CaseSearchRequest caseSearchRequest) {
         return createQueryString(boolQueryBuilder);
     }
 
+
+    // return must object
+    public Optional<QueryBuilder> createCaseType(String caseTypeId) {
```

Comment:
Could this be private?

Reply:
Can we rename function: e.g. `createFilterQueryForCaseType`